docs/painless/painless-guide/painless-execute-script.asciidoc

```diff
@@ -154,13 +249,560 @@ POST /_scripts/painless/_execute
     }
   }
 }
-----------------------------------------------------------------
+----
+// TEST[continued]
 
-Response:
+===== Response
 
 [source,console-result]
---------------------------------------------------
+----
 {
   "result": 0.8
 }
---------------------------------------------------
+----
+
+[[painless-execute-runtime-field-context]]
+===== Runtime field context
+The <<painless-runtime-fields-context,runtime field context>> runs a script used
+for a runtime field and returns results in doc-values order.
+
+Although runtime fields have their own context within the Painless execute API,
+you indicate the context for a runtime field based on the data type you want to
+return.
+
+[[painless-runtime-boolean]]
+====== `boolean_field`
+Use the `boolean_field` runtime field context when you want to return a `true`
+or `false` value from a script valuation. {ref}/boolean.html[Boolean fields]
+accept `true` and `false` values, but can also accept strings that are
+interpreted as either true or false. 
+
+Let's say you have data for the top 100 science fiction books of all time. You
+want to write scripts that return a boolean response such as whether books
+exceed a certain page count, or if a book was published after a specific year.
+
+Add the following fields to your mapping:
```

Comment:
If we're talking about this API as a way to test things without modifying the cluster it's probably worth rewording this to something more like "If you have an index like this:" .... "then you can run the execute API like this".